sqlmesh/core/config/model.py

```diff
@@ -45,6 +46,7 @@ class ModelDefaultsConfig(BaseConfig):
     on_destructive_change: t.Optional[OnDestructiveChange] = None
     session_properties: t.Optional[t.Dict[str, t.Any]] = None
     audits: t.Optional[t.List[FunctionCall]] = None
+    optimize: bool = True
```

Comment:
I think we wanna do `t.Optional[bool] = None`, otherwise you're always including bool in the payload unnecessarily. So `None` and `True` should lead to optimization, whereas an explicit `False` should not.